src/js/pages/Home.js

```diff
@@ -30,7 +30,8 @@ const i18nMessages = defineMessages({
 });
 
 const Home = ({ topServices, image, intl }) => {
-  const serviceLinks = cleanServiceLinks(topServices);
+  //TODO: clean data where sourced
```

Comment:
I don't understand the TODO

Reply:
oh, you mean we should expect Joplin to give us the full path in the query? If so, we should log this so Ben (or whoever) knows we want the query changed.

Reply:
Oh no, I'm meaning that data cleaning should happen in one place -- it could be Joplin, or maybe it happens in our react.static.config file. OR some other middleware layer for data transformations. But basically we should reformat data in one spot so when data formatting needs change, we only change it in one place AND so different components which render the same data in different place show that data consistently (or consume the same transformed data). But I'm not sure where this cleaning happens. for now, transformation consistency is taken care of since one shared method is performing the transform.